pkg/server/interceptors.go

```diff
@@ -43,12 +46,27 @@ func unaryInterceptor(ctx context.Context, req interface{}, info *grpc.UnaryServ
 	return resp, err
 }
 
-func combineInterceptors(a, b grpc.UnaryServerInterceptor) grpc.UnaryServerInterceptor {
-	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
-		return a(ctx, req, info, func(actx context.Context, areq interface{}) (interface{}, error) {
-			return b(actx, areq, info, func(bctx context.Context, breq interface{}) (interface{}, error) {
-				return handler(bctx, breq)
-			})
-		})
-	}
+// the always-yes logging decider function (because grpc_zap requires a decider function)
+func yesLogIt(ctx context.Context, fullMethodName string, servingObject interface{}) bool {
+	return true
+}
```

Comment:
is there a way to use this decider function instead of the if/else blocks in server.go, RE: https://github.com/storj/storj/pull/2067/files#diff-09856fe9becddf0199651f451409356aR48

Reply:
I don't think so, no. It looks like deciders have a different signature from interceptors. If the if/else blocks are particularly offensive, we could instead have replacements for grpc_middleware.ChainUnaryServer() and grpc_middleware.ChainStreamServer() which, if one of their arguments is nil, just returns the other argument unchanged.